apps/pl/scss/_scss2json.scss

```diff
@@ -29,13 +29,15 @@ $spacing-demo: $spacing;
 //@import '../_patterns/01-atoms/button/_button.scss';
 
 // Import the map of available custom font icons.
-@import '../../../source/_patterns/01-atoms/icon/scss/_icons-generated';
+@import '../../../source/_patterns/01-atoms/svgicon/scss/_icons-generated';
```

Comment:
Move `@import`s to top of file, generally.

Reply:
Leaving inline per pattern throughout this particular file.

Reply:
i think this one can move to the top, the icons won't conflict with any of bootstraps stuff. the other lower import for "customized variables" is specifically in the case that we _do_ override any bootstrap values (still could probably be moved to the top, underneath the main protons import)

Reply:
Moved to just below the bootstrap import and also cleaned up some commented out code from the old iconfont system as well in e22f35b.